streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -495,12 +496,9 @@ public void closeDirty() {
     }
 
     @Override
-    public void update(final Set<TopicPartition> topicPartitions, final ProcessorTopology processorTopology) {
-        super.update(topicPartitions, processorTopology);
+    public void update(final Set<TopicPartition> topicPartitions, final Map<String, List<String>> nodeToSourceTopics) {
```

Comment:
Could we add a unit test in `StreamTaskTest`?

Reply:
Ack, done